apps/api/src/interfaces/invoices.interface.ts

```diff
@@ -0,0 +1,38 @@
+enum InvoiceStatus {
+  Draft = 'draft',
+  Pending = 'pending',
+  Paid = 'paid',
+}
```

Comment:
Usually the convention for naming enums is with UPPER_CAMEL_CASE. Ref: https://stackoverflow.com/questions/13233137/javascript-enum-pattern-naming-convention
````javascript
enum InvoiceStatus {
  DRAFT = 'draft',
  PENDING = 'pending',
  PAID = 'paid',
}
````